src/gleam/dict.gleam

```diff
@@ -233,10 +233,10 @@ fn reverse_and_concat(remaining: List(a), accumulator: List(a)) -> List(a) {
   }
 }
 
-fn do_keys_acc(list: List(#(k, v)), acc: List(k)) -> List(k) {
+fn do_keys_loop(list: List(#(k, v)), acc: List(k)) -> List(k) {
```

Comment:
I assume the idea was to drop the do_ when there is a _loop?
Aka an oversight? @giacomocavalieri